applications/plugins/SofaPython/Binding_Data.cpp

```diff
@@ -747,4 +773,11 @@ SP_CLASS_ATTR(Data,name)
 SP_CLASS_ATTR(Data,value)
 SP_CLASS_ATTRS_END
 
-SP_CLASS_TYPE_BASE_PTR_ATTR(Data,BaseData)
+
```

Comment:
otherwise if there are two classes named like this in this namespace (which may happen) the linker will remove one and the constructor won't get called